test/Base_Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -167,6 +167,16 @@ spec_with suite_builder name create_new_datetime parse_datetime nanoseconds_loss
             dt1.zone . zone_id . should_equal interop_tz.zone_id
             dt1.to_display_text . should_equal "2020-01-01 01:00:00[+01:00]"
 
+        group_builder.specify "epoch in New York" <|
```

Comment:
This test isn't really related to this PR, but I don't want to create another PR just for it. The test verifies 
- #7655 

is fixed. It is by some upgrade of GraalVM.